src/components/FormManager.jsx

```diff
@@ -18,6 +19,19 @@ class FormManager extends React.Component {
     );
   };
 
+  //TODO: Add optional argument "isRequiredForCompleteness"
+  validateForm = () => {
+    const questions = this.context.getFormQuestionsData();
+
+    for (let i = 0; i < questions.length; i++) {
+      const question = questions[i];
+      ValidatorFactory.updateQuestionValidation(questions, question, i);
+      ValidatorFactory.updateSubQuestionsValidation(question);
```

Comment:
Not sure I like migration from FormUtils to ValidationFactory

Reply:
FormUtils is just set of utils so we did not try to obey single responsibility principle. While ValidatorFactory has one goal to create validator for a question ... while now, I do not know that much what is its responsibility 

-- but i did not think it too much through, so do not hesitate to ignore this comment.

Reply:
Maybe we can change the name to something like `QuestionValidationProcessor` or `ValidationProcessor`, so that it would put everything related to validation there?
My main motivation for moving from FormUtils was to remove a circular dependency.

Reply:
Cyclic dependency is good point.

Reply:
Yes I agree